src/lib.rs

```diff
@@ -1,21 +1,23 @@
-extern crate i2cdev;
+//! A simple library for using the Bosch BMP280 barometer and altimeter.
+//!
+//! This library has been tested on an Odroid-C4 using an [Adafruit BMP280](https://www.adafruit.com/product/2651) module.
+
+#![allow(dead_code)]
 extern crate byteorder;
-extern crate core;
+extern crate i2cdev;
 
-use i2cdev::linux::{LinuxI2CDevice, LinuxI2CError};
-use i2cdev::sensors::{Barometer, Thermometer};
```

Comment:
I see they removed these traits from `i2cdev`, that makes sense, it's quite odd for the core i2c library to have this abstraction in it.